docs/docker_container.md

```diff
@@ -120,7 +123,7 @@ Currently, the following versions are available:
 - 21.48.21782 - Ubuntu
 
 Example:
-```bash
+```
```

Comment:
Don't you need `bash` here?

Reply:
Previous test didn't include "Building an OpenVINO™ Model Server Docker Image" part. Should I add it now?

Reply:
reverted